### PR TITLE
chore(v3): add logging telemetry botaad driver

### PR DESCRIPTION
**Pull request by IvanJobs**
Add logging, telemetry, progressBar for botAadApp/create driver.
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/16034479

Working on increasing coverage.

### CODE REVIEW

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -1017,6 +1017,14 @@
   "configManager.error.driverNotFound": "Driver %s not found for step %s",
   "configManager.error.yamlParsing": "Error parsing yaml file at path %s. Reason: %s",
   "configManager.error.invalidLifecycle": "The schema of lifecycle %s is invalid",
+  "botRegistration.ProgressBar.creatingBotAadApp": "Creating bot aad app.",
+  "botRegistration.log.startCreateBotAadApp": "Creating bot aad app.",
+  "botRegistration.log.successCreateBotAadApp": "Bot aad app created successfully.",
+  "botRegistration.log.skipCreateBotAadApp": "Bot aad app creation skipped.",
```
Comment (blackchoey) [inline packages/fx-core/resource/package.nls.json:1023]: Not sure whether we want aligned wording (with minor changes) with aad app driver, can you sync with Ji on the wording?

Comment (IvanJobs) [reply packages/fx-core/resource/package.nls.json:1023]: Good suggestion. Will confirm wording with Ji.

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -1017,6 +1017,14 @@
...
+  "botRegistration.log.failCreateBotAadApp": "Failed to create bot aad app. Error message: %s.",
+  "driver.botAadApp.log.startExecuteDriver": "Executing driver %s",
```
Comment (jidddddd) [inline packages/fx-core/resource/package.nls.json:1025]: I think we are changing the name "driver" to "action", can you confirm @blackchoey ?

Comment (blackchoey) [reply packages/fx-core/resource/package.nls.json:1025]: Yes, I need to drive this change...

Comment (blackchoey) [reply packages/fx-core/resource/package.nls.json:1025]: @IvanJobs can you use `action` instead of `driver` in your PR first? thanks

**Review by eriolchan — DISMISSED** [review]

File: packages/fx-core/resource/package.nls.json [hunk 3]
```diff
@@ -1025,6 +1025,14 @@
...
   "configManager.error.invalidLifecycle": "The schema of lifecycle %s is invalid",
+  "botRegistration.ProgressBar.creatingBotAadApp": "Creating bot aad app.",
+  "botRegistration.log.startCreateBotAadApp": "Creating bot aad app.",
+  "botRegistration.log.successCreateBotAadApp": "Bot aad app created successfully.",
+  "botRegistration.log.skipCreateBotAadApp": "Bot aad app creation skipped.",
+  "botRegistration.log.failCreateBotAadApp": "Failed to create bot aad app. Error message: %s.",
+  "driver.botAadApp.log.startExecuteDriver": "Executing action %s",
+  "driver.botAadApp.log.successExecuteDriver": "Action %s executed successfully",
+  "driver.botAadApp.log.failExecuteDriver": "Failed to execute action %s. Error message: %s",
   "driver.aadApp.log.startExecuteDriver": "Executing driver %s",
```
Comment (Siglud) [inline packages/fx-core/resource/package.nls.json:1036]: We should use 'action 'instead of 'driver'

**Review by Siglud — APPROVED** [review]

**Review by hund030 — APPROVED** [review]

**Pull request merged**